client/my-sites/marketplace/components/reviews-cards/style.scss

```diff
@@ -0,0 +1,100 @@
+$border-color: #eee;
+
+.marketplace-reviews-cards__container {
+	position: relative;
+	display: flex;
+	flex-direction: column;
+	gap: 24px;
+
+	.marketplace-reviews-cards__reviews-title {
+		font-family: Recoleta, $sans;
+		color: var(--studio-gray-80);
+		font-size: $font-title-medium;
+	}
+
+	.marketplace-reviews-cards__reviews-subtitle {
+		font-family: "SF Pro Text", $sans;
+		color: var(--studio-gray-60);
+		font-size: $font-body-small;
+	}
+
+	.marketplace-reviews-cards__read-all {
+		font-family: "SF Pro Text", $sans;
+		font-size: $font-body-small;
+		position: absolute;
+		right: 0;
+		top: 25px;
+	}
+
+	.marketplace-reviews-cards__content {
+		display: flex;
+		gap: 20px;
+	}
+
+	.rating .rating__overlay .is-empty,
+	.rating .rating__star-outline .is-empty {
+		fill: var(--studio-yellow-20);
+	}
+
+	.rating .rating__overlay .gridicon,
+	.rating .rating__star-outline .gridicon {
+		fill: var(--studio-yellow-20);
+	}
+}
+
+
+.marketplace-reviews-card__container {
+	padding: 24px 30px;
+
+	display: flex;
+	flex-direction: column;
+	gap: 16px;
+}
+
+.marketplace-reviews-card__container,
+.marketplace-reviews-card__leave-a-review {
+	border: 1px solid $border-color;
+	border-radius: 4px;
+	width: 50%;
```

Comment:
This is actually using more than half of the available space, but it shrinks to the right size due to Flexbox.

This needs investigation as it becomes a problem when only one card is displayed.

Reply:
Fixed here 369ff1ebcb96f0a8ebc52e94ffdf65265dbfcc35 by using border-box

Reply:
> by using border-box

Such a classic 🤓, I think we should have a reset with that for new pages, but I understand we don't want to create a big regression impact